Control-Tower-For-Horizon/lambda/register_new_horizon_account_v2.py

```diff
@@ -0,0 +1,231 @@
+import json
+import logging
+import os
+import sys
+import boto3
+import requests
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# CONSTANTS
+SUCCESS = "SUCCESS"
+FAILED = "FAILED"
+
+
+CSPM_ROLE_TEMPLATE_URL = 'https://cs-prod-cloudconnect-templates.s3-us-west-1.amazonaws.com' \
+                         '/aws_cspm_cloudformation_v1.1.json'
+STACK_SET_NAME = "CrowdStrike-CSPM-Integration"
```

Comment:
Can this be parameterized to ensure the templates are pulled from within a region? Eg workloads running in GovCloud may not want to connect to other regions.